### ISSUE TITLE
[Staging Beta 7.0.0 #D2423858] Crashing on updating - "System.Exception: Error(s) occurred when loading Users"

### ISSUE THREAD

**Issue by pamduncan** [StrangeLoopGames/EcoIssues#3766]
I did a clean install of Staging Beta 7.0.0 #D2423858 but am using and existing world.  The server started crashing over and over.  Logs attached.  I checked the users in the Game.eco and there were 4 users names 'slg###".  ALso there was a file in the storage folder called "Default World" and it contained only a few folders - not a complete save.

I deleted the 'default world' and I also removed the users designated slg#### and restarted the server.  It loaded normally and is now up an running.  Were those 'slg' users ones in the process of being converted for this staging and conversion failed and I have now lost the user?

https://www.dropbox.com/s/r8ahv6odlwj0qcf/Dump01.zip?dl=0

Still getting exception thrown if DetectNAT is set to true (reported in previous staging)

**Comment by MarjieAVolk** [comment]
Yeah you're not supposed to delete Default World.  @metachronism what will even happen in that case?  Maybe that file should be located elsewhere, since people are used to having free reign over the storage folder.

Users named slg# are expected now, as we have changed some user identification things due to steam integration.  I wish they were still named their display name in the save, but I am not sure how to do so.  @Nanders did you change this on purpose?  We should be able to use display name still, since we've enforced that they are unique.

So sounds like there was some kind of problem during your save file migration.  I would imagine that deleting the users fixed the problem, rather than deleting the default world.  What version was your save on before you migrated it to the current staging?
